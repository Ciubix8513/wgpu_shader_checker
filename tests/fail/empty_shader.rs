use wgpu_shader_checker::include_wgsl;

fn main() {
    include_wgsl!("../../../../shaders/empty.wgsl");
}
