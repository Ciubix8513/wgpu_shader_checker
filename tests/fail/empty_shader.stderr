error: ../../../../shaders/empty.wgsl is empty
 --> tests/fail/empty_shader.rs:4:5
  |
4 |     include_wgsl!("../../../../shaders/empty.wgsl");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `include_wgsl` (in Nightly builds, run with -Z macro-backtrace for more info)
